engine/configstore/supabaseLegacy.ts

```diff
@@ -67,67 +88,83 @@ interface AudienceFlag {
   __resolveType: string;
 }
 
-interface CatchAllConfigs {
-  handler: {
-    flags: AudienceFlag[];
-    __resolveType: "$live/handlers/routesSelection.ts";
-  };
-  __resolveType: "resolve";
-}
-
 const sectionToAccount: Record<string, string> = {
   "deco-sites/std/sections/configVTEX.global.tsx":
     "deco-sites/std/accounts/vtex.ts",
   "deco-sites/std/sections/configOCC.global.tsx":
     "deco-sites/std/accounts/occ.ts",
   "deco-sites/std/sections/configShopify.global.tsx":
     "deco-sites/std/accounts/shopify.ts",
+  "deco-sites/std/sections/configVNDA.global.tsx":
+    "deco-sites/std/accounts/vnda.ts",
+  "deco-sites/std/sections/configYourViews.global.tsx":
+    "deco-sites/std/accounts/yourViews.ts",
+};
```

Comment:
these are the wellKnown global sections that is supposed to be accounts instead.